quickstarts/aws/aws-s3/alerts/HighErrorCount.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: High Error Count
```

Comment:
@EmilHammarstrand Thanks for adding this alert. Is their an opportunity to add any more dashboards for this quickstart?

Reply:
@jpvajda Adding more alerts? :)

Reply:
@EmilHammarstrand I was thinking if any additional dashboards would make sense, or additional alerts to increase the value of this quickstart. This may be enough for now however. Just let me know if you feel like any additions would be worth doing.

Reply:
@jpvajda Sure!

Reply:
@EmilHammarstrand Thank you!